dts/bindings/display/raydium,rm68200.yaml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2022, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: Raydium RM68200 Panel
+
+compatible: "raydium,rm68200"
+
+include: mipi-dsi-device.yaml
+
+properties:
+    label:
+      required: true
+
+    reset-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The RESETn pin is asserted to disable the sensor causing a hard
+        reset.  The sensor receives this as an active-low signal.
+
+    pwr-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The PWRn pin is asserted to do a power on for the panel.
+        The panel receives this as an active-high signal.
```

Comment:
why not use the regulator API here? Define a regulator that handles display power, and, if displays needs to control power manually make use of it.

Reply:
I don't much of a benefit in using regulator API here.

Reply:
> I don't see much of a benefit in using regulator API here.

Please explain why not.

Reply:
The use is fairly simple which is to enable a GPIO line. Hence my thought if there was a need to got through the regulator API.